scripts/winesapos-install.sh

```diff
@@ -787,6 +789,16 @@ elif [[ "${WINESAPOS_DE}" == "gnome" ]]; then
         pacman_install_chroot manjaro-gnome-settings manjaro-settings-manager
     fi
     echo "Installing the GNOME desktop environment complete."
+elif [[ "${WINESAPOS_DE}" == "i3" ]]
+      echo "Installing i3"
```

Comment:
nit: there is an extra whitespace behind `echo`. It should align with the code below.